lib/rubocop/cop/lint/ensure_return.rb

```diff
@@ -26,7 +26,9 @@ module Lint
       #     do_something_else
       #   end
       class EnsureReturn < Cop
-        MSG = 'Do not return from an `ensure` block.'.freeze
+        MSG = 'Do not return from an `ensure` block, it changes the control ' \
+              'flow as if a `rescue Exception` clause was in place before ' \
+              'the `ensure` clause.'.freeze
```

Comment:
As I looked at [the discussion comment](https://github.com/rubocop-hq/rubocop/issues/5949#issuecomment-394906395).  it seems better to change [the class document](https://github.com/rubocop-hq/rubocop/pull/6064/files#diff-40ffaa98b1eb2bf3213e7febd2eeca07R6) rather than this offense message.

I think that long offense message is not favored.
https://github.com/rubocop-hq/rubocop/pull/5845#issuecomment-385845141

Reply:
Ok, make sense. Thank you, will change this

Reply:
And add an entry in the style guide and link to it from the cop's metadata.